test/uk/gov/hmrc/cgtpropertydisposals/models/Generators.scala

```diff
@@ -419,14 +407,12 @@ trait AddressGen extends AddressLowerPriorityGen {
   }
 
   implicit val countryGen: Gen[Country] = Gen.oneOf(Country.countryCodes.map(Country(_)))
-
 }
```

Comment:
I think those changes fit more properly under the cosmetics/refactoring rather than Play 3.0 upgrade but its just a suggestion for a future

Reply:
No probs, will try and keep them separate in the future